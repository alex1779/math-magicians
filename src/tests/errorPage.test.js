import renderer from 'react-test-renderer';
import React from 'react';
import ErrorPage from '../components/Errorpage';

test('Confirm that ErrorPage renders correctly', () => {
  const component = renderer.create(<ErrorPage />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
