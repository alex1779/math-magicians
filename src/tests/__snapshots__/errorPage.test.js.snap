// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirm that ErrorPage renders correctly 1`] = `
<div
  className="error"
>
  <h2
    className="h2"
  >
    Ups...Something went wrong!
  </h2>
  <p
    className="p"
  >
    404 not found
  </p>
</div>
`;
